### PR TITLE
ReaderService  system implementation is done

### PR DESCRIPTION
**Pull request by andromash**

### CODE REVIEW

File: src/main/java/spring/boot/util/SystemFileReader.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@Component
+public class SystemFileReader implements ReaderService {
+
+    @Override
+    public List<String> readData(String dataSource) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(dataSource))) {
+            return reader.lines().collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new RuntimeException("Can not read data from file with path " + dataSource, e);
```
Comment (AnaLanda) [inline src/main/java/spring/boot/util/SystemFileReader.java]:
```suggestion
            throw new RuntimeException("Cannot read data from the file with the path " + dataSource, e);
```

**Review by AnaLanda — COMMENTED** [review]
Looks great! 😃

File: src/test/java/spring/boot/util/SystemFileReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    private static final String EMPTY_FILE_NAME = "/emptyfile.csv";
+    private static final String WRONG_PATH = "/magic.csv";
+
+    @Autowired
+    private ReaderService readerService;
+
+    @Test
+    public void readFileOk() {
+        List<String> expected = List.of("1,test,superb,no logic here, just reading",
+                "2,test2,super,no logic here, just reading");
```
Comment (dmytro-hryhoruk) [inline src/test/java/spring/boot/util/SystemFileReaderTest.java]:
```suggestion
        List<String> expected = List.of("1, test, superb, no logic here, just reading",
                "2, test2, super, no logic here, just reading");
```

**Review by dmytro-hryhoruk — COMMENTED** [review]
set up travis and checkstyle plugin in this project

File: src/test/java/spring/boot/util/SystemFileReaderTest.java (hunk 2 above)
Comment (andromash) [reply src/test/java/spring/boot/util/SystemFileReaderTest.java]: is it obligatory to insert spaces in csv?

**Comment by andromash** [comment]
Travis configuration is in the task of hw-5. Configured, but better change the task :)

File: src/test/java/spring/boot/util/SystemFileReaderTest.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/test/java/spring/boot/util/SystemFileReaderTest.java]: not it is not)you may leave it

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**